nf_core/pipelines/rocrate.py

```diff
@@ -336,6 +348,28 @@ def add_main_authors(self, wf_file: rocrate.model.entity.Entity) -> None:
             if author in authors:
                 wf_file.append_to("maintainer", author_entitity)
 
+    def update_rocrate(self) -> bool:
+        """
+        Update the rocrate file
+        """
+        # check if we need to output a json file and/or a zip file based on the file extensions
+        json_path = None
+        zip_path = None
```

Comment:
```suggestion
        json_path = None
        zip_path = None
```
these are actually not needed